megatron/neox_arguments/neox_args.py

```diff
@@ -343,6 +343,26 @@ class NeoXArgsModel(NeoXArgsTemplate):
       x = x + attn(y) + mlp(y)
     """
 
+    use_bias_in_norms: bool = True
+    """
+    If false, norms (e.g. LayerNorm) will not have bias terms
+    """
+    use_bias_in_attn_linear: bool = True
+    """
+    If false, attn_linear (e.g. QKVO) will not have bias terms
+    """
+
+    mlp_type: str = "regular"
+    """
+    Types:
+        regular: Megatron implementation
+        llama: LLaMA MLP
+    """
+    llama_mlp_multiple_of: int = 256
+    """
+    LLaMA MLP sizes are padded to a nice number
```

Comment:
Is this only applicable to llama MLPs? Can we add this functionality to the codebase as a whole?

Reply:
Actually, I might take this out. This is a constraint for how they use to compute the intermediate dimension, but we can just hard-code it into the configuration for easier readability.

Reply:
Okay, I think that makes sense then. So we’d have a “llama” config file which satisfies this, but not tell the user to follow it in general?